### DISCUSSION TITLE
Is a Renovate Pro Commercial License vaild for Renovate EE?

### DISCUSSION THREAD

**Discussion by dkr91** [mend/renovate-ce-ee#407] [Q&A] [answered]
Hey, I was wondering if the Old Keys for Renovate Pro are still valid to be used by Renovate EE?

**Comment by rarkins** [comment, marked as answer]
Renovate Pro was discontinued as a commercial product in 2019. It was renamed to Renovate On-Prem and made free, and any paid license key issued before then remained valid for the new, free Renovate On-Prem.

In 2023 Renovate On-Prem was rewritten and renamed to Renovate CE, remaining free, and all old keys remain valid. Additionally Renovate EE was added, as a new commercial software with annual license and its own license key system.

Therefore:
 - Unless you purchased in 2018-2019, you never had a commercial license
 - Even if you did, it expired after 12 months from the last time you paid
 - Renovate EE is a new product and any licenses for it need to be purchased from 2023 onwards
